### DISCUSSION TITLE
Programming Discord Bot on Repl.it to send an Image File

### DISCUSSION THREAD

**Discussion by Skigamer13** [Rapptz/discord.py#6798] [Q&A] [answered]
I'm having some trouble coding a discord bot to send an image file on Repl.it in Python, I already have a basic outline of the bot code, I just need help with having the bot send an image on command.

**Comment by apple502j** [comment, marked as answer]
First, why are you running it on repl.it? It's much better to have your laptop run it 24/7.

You should also check the [FAQ](https://discordpy.readthedocs.io/en/master/faq.html#how-do-i-upload-an-image).

**Reply by Skigamer13** [reply]
thx for the help!
